harvester_integration_test.go

```diff
@@ -23,6 +23,7 @@ var (
 		`INFO: automatically monitoring consul key "harvester/age"`,
 		`INFO: automatically monitoring consul key "harvester/balance"`,
 		`INFO: automatically monitoring consul key "harvester/has-job"`,
+		`INFO: automatically monitoring consul key "harvester/FunTime"`,
```

Comment:
Maybe try keeping the keys consistent for readability and not use camel case (`harvester/fun-time`)